### ISSUE TITLE
DLL error prevents cellpose opening

### ISSUE THREAD

**Issue by maryebrown1** [MouseLand/cellpose#505]
I get the following error when I try to open cellpose:

`GUI ERROR: DLL load failed while importing QtGui: The specified module could not be found.
GUI FAILED: GUI dependencies may not be installed, to install, run
     pip install cellpose[gui]`

First I installed cudatoolkit, cudnn, and tensorflow. Then I installed cellpose exactly as described in the instructions on the GitHub page. Do I need to reinstall QtGui? I can't find a python command for installing QtGui.

**Comment by carsen-stringer** [comment]
tensorflow is not required for cellpose, we use pytorch. what is your operating system? did you install the full version of cellpose with the gui? `pip install cellpose[gui]` or `pip install cellpose[all]`?

also did you use a new environment in anaconda? that could help in case conflicting versions of pyqt were installed (`conda create --name cellpose python=3.8`)

**Comment by carsen-stringer** [comment]
I'm guessing this is in windows? I also use windows and do not have this error, here are my package versions:
```
pyqt5                     5.15.6                   pypi_0    pypi
pyqt5-qt5                 5.15.2                   pypi_0    pypi
pyqt5-sip                 12.9.0                   pypi_0    pypi
pyqtgraph                 0.11.0rc0                pypi_0    pypi
```

**Comment by maryebrown1** [comment]
Thank you for your help. I ended ip going to IT help with the College of Science and Engineering at Univ. of Minnesota - Twin Cities and we figured out that Tensorflow was not required and we needed pytorch. Now we have the cellpose GUI working and using the GPU. I'll verify our pyqt versions. We made a new environment in Anaconda Navigator and installed cellpose with 

> python -m pip install cellpose[all]

.Thanks again for getting back to me.
